### DISCUSSION TITLE
Gas Meter live consumption not working Home Mini

### DISCUSSION THREAD

**Discussion by x13pow** [BottlecapDave/HomeAssistant-OctopusEnergy#979] [Q&A]
Hello, i have a home mini which seems to be working great but my gas meter doesn't show live readings of consumption, the entity is there but doesn't work, has anyone else had this?

**Comment by BottlecapDave** [comment]
Hello. Because the gas meter is battery based, the data isn't live but is updated more frequently (roughly every 30 minutes). This also makes it more prone for the mini to lose connection. Do you have access to the data in the app (for today)?

**Reply by x13pow** [reply]
Thank you for the reply Dave! here is all the entities
![image](https://github.com/user-attachments/assets/0739317f-2371-4d19-8316-2db1efc03531)

**Reply by BottlecapDave** [reply]
Your current gas accumulative consumption sensors appear to have data (the top two sensors in your screenshot). The current consumption (4th one down) will be the delta reading since the last reading. Because of the frequency of data updates for gas, this will be 0 the majority of the time. But if you look at the history, you should see spikes.
